retriever/rt_dictionary.py

```diff
@@ -1,6 +1,6 @@
-from arguments import Arguments
-from dictionary import Dictionary
-from logger import Logger
+from retriever.arguments import Arguments
+from utils.dictionary import Dictionary
+from utils.logger import Logger
```

Comment:
Ditto

Reply:
Maybe
from retriever.arguments import Arguments 
it could be written as 
from .arguments import Arguments
because is in the same subpackage
But I guess that the other ones can be expressed with the subpackage where they are defined